draft-birkholz-scitt-receipts.md

```diff
@@ -128,17 +154,24 @@ The following parameters MUST be included in the protected header of the counter
 
 # CCF 2 Tree Algorithm
 
-The CCF 2 tree algorithm corresponds to the algorithm implemented in the CCF version 2 framework.
+The CCF 2 tree algorithm documents the algorithm based on a binary Merkle tree over the sequence of all ledger entries that is implemented in the CCF version 2 framework.
+
+> Add pointer to CCF v2? 
```

Comment:
https://microsoft.github.io/CCF/main/architecture/merkle_tree.html

Reply:
Let me look up what the syntax is to include non-normative references like that.

Reply:
Something like this should work at the top, reference with `{{CCF_Merkle_Tree}}`:
```
informative:
  CCF_Merkle_Tree:
    target: https://microsoft.github.io/CCF/main/architecture/merkle_tree.html
    title: CCF - Merkle Tree
    author:
      ins: Microsoft Research
```